### PR TITLE
Add container runtime and image projects

### PR DESCRIPTION
**Pull request by mheon**

### CODE REVIEW

**Comment by rhatdan** [comment]
I think you should add containers/storage and containers/image or is this just for actual products?

File: app/data/projects.json [hunk 1]
```diff
@@ -66,13 +66,31 @@
...
   },
+  {
+    "projectName": "containers/image",
+    "projectDescription": "A library for working with container images",
+    "projectRepository": "https://github.com/containers/image",
+    "category": "Development"
+  },
+  {
+    "projectName": "containers/storage",
+    "projectDescription": "A library which for storing and mounting container images",
```
Comment (robin-owen) [inline app/data/projects.json]: Typo: "which for"

Comment (mheon) [reply app/data/projects.json]: Fixed

**Comment by mheon** [comment]
Added one more per suggestion (Moby project)

**Pull request merged**